it/mail/chrome/messenger/preferences/applications.dtd

```diff
@@ -24,14 +24,14 @@ entities below were automatically added by l10nmerge
 -->
 
 <!ENTITY enableCloudFileAccountOffer.mb "MB">
-<!ENTITY authRequired.button.label        "Authorize">
+<!ENTITY authRequired.button.label        "Autorizza">
 <!ENTITY authRequired.button.accesskey    "U">
 <!ENTITY addCloudFileAccount.accesskey    "A">
-<!ENTITY removeCloudFileAccount.label     "Remove">
-<!ENTITY addCloudFileAccount.description  "Add a new online storage service">
-<!ENTITY authRequired.description         "Your authorization is required in order to see the settings for this storage service.">
-<!ENTITY addCloudFileAccount.label        "Add">
+<!ENTITY removeCloudFileAccount.label     "Rimuovi">
+<!ENTITY addCloudFileAccount.description  "Aggiungi un nuovo servizio di archiviazione online">
+<!ENTITY authRequired.description         "È richiesta la tua autorizzazione per visualizzare le impostazioni per questo servizio di archiviazione.">
+<!ENTITY addCloudFileAccount.label        "Aggiungi">
 <!ENTITY removeCloudFileAccount.accesskey "R">
-<!ENTITY attachments.incoming.label   "Incoming">
-<!ENTITY attachments.outgoing.label   "Outgoing">
-<!ENTITY enableCloudFileAccountOffer.label "Offer to share for files larger than">
+<!ENTITY attachments.incoming.label   "In entrata">
+<!ENTITY attachments.outgoing.label   "In uscita">
+<!ENTITY enableCloudFileAccountOffer.label "Offerta di condivisione pe file maggiori di">
```

Comment:
CORRECTIONS NEEDED. A letter is missing in one word ("per" instead of "pe"). The correct sentence should be as follows: 

 +<!ENTITY enableCloudFileAccountOffer.label "Offerta di condivisione per file maggiori di">